specification/compute/resource-manager/Microsoft.Compute/stable/2020-05-01/examples/GetInformationAboutDiskAccess.json

```diff
@@ -24,7 +24,9 @@
                   }
                 }
               }],
-            "timeCreated": "2020-03-10T18:09:06.3801919+00:00",
+            "privateEndpoint": {
```

Comment:
Why do we need a separate class with a single member, instead of 
"privateEndpointId" : "/subscriptions" etc?

Reply:
this was given by NRP in their API spec so that for Portal there are no specific changes needed for DiskAccess